pkg/stis/calstis/cs1/dodqi.c

```diff
@@ -652,6 +653,8 @@ int doppmin, doppmax  i: offsets for Doppler shift
 	    xlength = 2;
 	    ylength = tabrow->length * 2;
 	}
+	else
+	    assert(0);
```

Comment:
assert(0) isn't a graceful exit, I believe it calls abort() (as printed above) which doesn't do cleanup. abort core dumps are not a great way to go. I think we  should minimally set the appropriate return flag and report the problem instead with a useful error statement, and add any cleanup code which is necessary for termination.